packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -22,6 +26,36 @@ class AdaptiveCard extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps !== this.props && this.props.verb === API_ACTIVITY_VERB.SHARE) {
```

Comment:
Can you add a snapshot test for an adaptive card with blob URLs to tests this changes?

Reply:
Please refer this https://github.com/webex/react-widgets/pull/1014#discussion_r339013901. As discussed with @adamweeks if we are not able to do in snapshot testing , will check in journey tests